packages/react-positioning/src/createArrowStyles.ts

```diff
@@ -1,80 +1,102 @@
-import { shorthands } from '@fluentui/react-make-styles';
-import type { MakeStylesStyleRule } from '@fluentui/react-make-styles';
+import { MakeStylesStyle, shorthands } from '@fluentui/react-make-styles';
 import type { Theme } from '@fluentui/react-theme';
 
+/**
+ * Options parameter for the createArrowStyles function
+ */
+export type CreateArrowStylesOptions = {
+  /**
+   * The height of the arrow from the base to the tip. The base width of the arrow is always twice its height.
+   *
+   * If this is omitted, you must add styles created by createArrowHeightStyles to set the arrow's size correctly.
+   */
+  arrowHeight?: number;
+
+  /**
+   * The border of the arrow. This should be the same border as the parent element.
+   */
+  border?: [/*width:*/ string, /*style:*/ string, /*color:*/ string];
```

Comment:
Can we use separate object properties there, i.e. `borderWidth`, `borderStyle` & `borderColor` separately?

```suggestion
  borderColor?: MakeStylesStyle['borderColor'],
  borderStyle?: MakeStylesStyle['borderStyle'],
  borderWidth?: MakeStylesStyle['borderWidth']
```

(👆 _didn't test it, but should work_)

- it will be more readable
- removes need to have comments to describe the tuple
- is not order dependent
- as styles are compiled anyway - does not have any effect on bundle size